ddtrace/internal/telemetry/writer.py

```diff
@@ -105,6 +109,8 @@ def _reset_queues(self):
             self._events_queue = []
 
     def periodic(self):
+        self.app_heartbeat_event()
```

Comment:
I am not sure if this has changed in the past few months but we should only send a heartbeat event if the telemetry events queue is empty (ie we should not send a heartbeat event with app-started, app-integration-changed and app-closed).

Reply:
This should fix the failing telemetry test. I'll make this change